### DISCUSSION TITLE
faq/

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#158] [Announcements]
# faq/

Catalog How to ... (howto.md); Why ... (why.md); See also ... (seealso.md);

https://alist.nn.ci/faq/

**Comment by chensigong** [comment]
请问挂在了百度网盘，但是进去以后报错了failed get objs: failed to list objs: errno: -7, refer to https://pan.baidu.com/union/doc/，不知道怎么解决

**Comment by yzx0511** [comment]
一直显示请求过多 怎么解决

**Comment by Has119** [comment]
115网盘不能复制文件进去了吗？一直显示错误

**Comment by TroubleMakerZ** [comment]
TRUENAS 云同步任务 向ALIST的webdav里的百度云盘 推送同步数据  经常会出现 Failed to copy: Method Not Allowed: 405 Method Not Allowed

**Comment by pkids** [comment]
希望能实现自动判断挂在的网盘是否因为cookies有效期到了等原因导致的掉盘，然后发邮件告诉我重新去修改参数重新绑定，不然cloud sync有时候无法同步，等到我发现了以后，已经很多数据没同步到网盘了，就算重新修改了参数绑定了网盘，哪些没同步上去的数据也无法自动再同步上去了。。。。
